### ISSUE TITLE
Create a Sound Manager

### ISSUE THREAD

**Issue by jedeen** [excaliburjs/Excalibur#590]
### Context
It would be incredibly helpful to manage sounds in a central location. 
##### Features
- global volume management of all sounds (see also #265)
- grouped-by-type volume management (see also #380)
  - built-in collections (music, sound effects)
  - support for custom collections and sub-collections
- preserve and balance individual sound levels

**Comment by alanag13** [comment]
I've talked about this before; being able to increase / decrease volume of sounds / sound groups by a  percentage rather than set them to a given value would help to preserve balance.

**Comment by kamranayub** [comment]
Yes, I think organizing by the typical "tracks" and allowing track volume setting should help solve this and make it easier to manage sounds.
